APE24.rst

```diff
@@ -0,0 +1,477 @@
+the pyOpenSci form above, specifying:
+
+* astronomy domain expertise, and
+* membership in the Astropy community.
+
+pyOpenSci does not publicly list all the people who signed up
+(i.e, a reviewer is only public during the review process)
+but the `pyOpenSci Editorial Board`_ has access to the list.
+
+pyOpenSci will respect the Astropy-specific criteria for someone to be a
+reviewer for Affiliated request, namely:
+
+* familiarity with the Astropy project,
+* ability to judge whether a package integrates well with the Astropy ecosystem
+  (as per pre-APE 24 guidelines), and
+* having domain expertise in the area of the package (e.g., galaxy evolution).
+
+pyOpenSci and Astropy both ask editors/reviewers disclose any potential conflict of interest (COI)
+prior to agreeing to review a package. In the event where COI occurs, with this partnership,
+we would follow the
+`pyOpenSci COI process <https://www.pyopensci.org/software-peer-review/our-process/policies.html#conflict-of-interest-for-reviews-and-editors>`_.
+pyOpenSci invites the Astropy community to review this language to ensure it meets our needs.
+
+.. _reviewer sign-up form: https://forms.gle/GHfxvmS47nQFDcBM6
+
+Review turnaround time
+----------------------
+
+Astropy currently does not enforce any concrete turnaround time mainly because
+we do not know who ends up reviewing a given submission nor have control
+over their schedule. Furthermore, we only have two editors who may not have
+sufficient time themselves to actively push a review forward. Therefore, a review
+could take weeks, months, or (gasp) years.
+
+If we partner with pyOpenSci, Astropy would need to adhere to their expected
+timeline for each step, as laid out in `An Overview Of the Peer Review Process`_.
+For example, editor is expected to find reviewers within 2-3 weeks and
+a peer review should be completed within 3 weeks after that.
+This is to ensure that the package maintainer have a good experience with the review
+and things do not languish over a long period of time.
+
+Generally, the editor role should not take a huge amount of time, *but* it is
+important for an editor, once the review starts, to check in on the review periodically
+(every few weeks and more often during wrap-up).
+
+.. _An Overview Of the Peer Review Process: https://www.pyopensci.org/software-peer-review/our-process/review-timeline.html
+
+
+Packaging guidelines
+--------------------
+
+pyOpenSci has been developing a `community-driven packaging guide`_
+that covers modern best practices and recommendations for scientific Python packaging.
+This guide has a stringent community review process. Reviewers for this guide
+represent members of PyPA, core Python, Anaconda (conda/conda-forge), and
+core packages for front- and back-end tools (e.g., ``flit``, PDM, ``hatch``/``hatchling``, etc.).
+
+Meanwhile, the Astropy community prefers a more laid-back
+`OpenAstronomy packaging guide <https://packaging-guide.openastronomy.org/en/latest/>`_.
```

Comment:
```suggestion
Packaging guidelines
--------------------
The Astropy community has followed packaging guidelines published in the 
`OpenAstronomy packaging guide <https://packaging-guide.openastronomy.org/en/latest/>`_ for XX years.

pyOpenSci is also developing a `community-driven packaging guide`_
that covers modern best practices and recommendations for scientific Python packaging. 
This guide has a stringent community review process. Reviewers for this guide
represent members of PyPA, core Python, Anaconda (conda/conda-forge), and
core packages for front- and back-end tools (e.g., ``flit``, PDM, ``hatch``/``hatchling``, etc.).  

If we partner with pyOpenSci, we encourage our Affiliated packages to continue to follow the OpenAstronomy packaging guide for packaging structure. However maintainers will find the pyOpenSci guide to be a useful reference given it provides community-wide guidelines and is targeted to those newer to packaging. 

```

or something to that effect. so we aren't completely deleting our guide but making sure folks know the difference?

Reply:
@Cadair , does this change satisfy you?